pemutil/pem.go

```diff
@@ -707,3 +708,30 @@ func ParseSSH(b []byte) (interface{}, error) {
 		return nil, errors.Errorf("unsupported key type %T", key)
 	}
 }
+
+// AppendToBundle adds a PEM-encoded certificate to a PEM-encoded certificate
+// bundle if not already in the bundle.
+func AppendToBundle(bundlePEM, certPEM []byte) ([]byte, bool, error) {
```

Comment:
A variadic method might be better:
```go
func AppendToBundle(bundlePEM []byte, certsPEM ...[]byte) ([]byte, bool, error)
```

Reply:
And perhaps rename it to `BundleCertificate`, something related to certificates (better than `AppendtoCertificateBundle`. The pemutil package manages keys too.

Reply:
a084886